### PR TITLE
Added social dominance query and schema for it

### PR DESCRIPTION
**Pull request by valo**
Broke down the social dominance query into a separate file, as the main
schema is getting out of hand.

### CODE REVIEW

File: lib/sanbase_web/graphql/schema.ex [hunk 1]
```diff
@@ -585,6 +586,8 @@ defmodule SanbaseWeb.Graphql.Schema do
       resolve(&TechIndicatorsResolver.social_volume/3)
     end
 
+    import_fields :social_data_queries
```
Comment (IvanIvanoff) [inline lib/sanbase_web/graphql/schema.ex]: :tada:

File: lib/sanbase_web/graphql/schema/social_data_queries.ex [hunk 2]
```diff
@@ -0,0 +1,37 @@
+defmodule SanbaseWeb.Graphql.Schema.SocialDataQueries do
+  use Absinthe.Schema.Notation
+
+  alias SanbaseWeb.Graphql.Resolvers.TechIndicators
+  alias SanbaseWeb.Graphql.Complexity
+  alias SanbaseWeb.Graphql.Middlewares.ApiTimeframeRestriction
+
+  import_types(SanbaseWeb.Graphql.SocialDataTypes)
```
Comment (IvanIvanoff) [inline lib/sanbase_web/graphql/schema/social_data_queries.ex]: Will this work if you have the `import_types` in `schema.ex`? I am not sure how well this approach will work when we need the same types in multiple `Queries` modules

Comment (valo) [reply lib/sanbase_web/graphql/schema/social_data_queries.ex]: It won't work. That's why this change: https://github.com/santiment/sanbase2/pull/1098/files#diff-e77f0e25b129cf9d83250e9c1ef10d6cL58

We will need to take care of circular dependencies somehow. If certain types are shared we might not be able to break the queries into multiple files.

File: lib/sanbase/social_data/social_dominance.ex [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    end
+  end
+
+  defp social_dominance_request(
+         datetime_from,
+         datetime_to,
+         interval,
+         source
+       ) do
+    cache_key = Cache.cache_key(:api_request, %{
```
Comment (IvanIvanoff) [inline lib/sanbase/social_data/social_dominance.ex]: `:social_dominance_api_request` to include the name of the query

**Comment by IvanIvanoff** [comment]
@also ru the formatter

File: test/sanbase/social_data/social_dominance_test.exs [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+    [
+      project: project
+    ]
+  end
+
+  describe "social_dominance/5" do
+    test "response: success" do
+      from = 1_523_876_400
+      to = 1_523_880_000
```
Comment (IvanIvanoff) [inline test/sanbase/social_data/social_dominance_test.exs]:
```
from = DateTime.from_naive!(~N[2018-04-16 10:00:00], "Etc/UTC")
to = DateTime.from_naive!(~N[2018-04-16 22:00:00], "Etc/UTC")
```
and you can drop the `from_unix!`

**Comment by valo** [comment]
Ran the format

**Pull request merged**